mopro-ffi/src/app_config/android.rs

```diff
@@ -2,9 +2,9 @@ use std::fs;
 use std::path::Path;
 use std::process::Command;
 
-use super::{cleanup_tmp_local, install_arch, install_ndk, mktemp_local};
+use uniffi::{generate_bindings, KotlinBindingGenerator};
```

Comment:
After I wrote this rust code I started separating imports by line. It makes the file a bit more clean IMO and more resistant to conflicts. Totally optional

```suggestion
use uniffi::generate_bindings;
use uniffi::KotlinBindingGenerator;
```

Reply:
I'd love to automate this, is there a linter for that?